tools/plugins/com.liferay.ide.core/plugin.xml

```diff
@@ -77,6 +77,15 @@
       </runtime>
    </extension>
    <extension
+         id="com.liferay.ide.core.liferayGradleWebNature"
```

Comment:
if we don't want to add a new nature, then we need to check gradle mode to check whether the project apply liferay theme build plugin, I remembered that you said check gradle model will be slow every time. So I find this solution.